src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx

```diff
@@ -0,0 +1,66 @@
+export const meta = {
+  title: 'Monitoring with Playwright on Checkly made easy',
+  description: "Use Playwright to run your status checks in the cloud to ensure the high availability of your web applications.",
+  author: "Max Schmitt",
+  date: "2020-09-23",
+  tags: ["checkly", "monitoring", "playwright"],
+  hidden: true
+}
+
+## Introduction
+
+Having a safe, functional, and feature-rich configured monitoring and status checks in place is nowadays needed to ensure a fast error reporting and by that a good uptime so your SLA is getting respected. The typical monitoring solutions are e.g. Grafana which would be feature-rich but has no good native integration for launching browsers to emulate a real user. Also, you need to host it yourself which is mostly for monitoring solutions not advised (or you need a HA installation in another data center or cloud provider).
```

Comment:
My suggestion would be to break down the first phrase, or to rephrase to allow for better punctuation.
One example could be: "Nowadays, catching issues with your service early on is paramount to ensuring that your SLAs are being met. For this reason, having a reliable and feature-rich active monitoring solution in place is a must."

Reply:
Good point. thx